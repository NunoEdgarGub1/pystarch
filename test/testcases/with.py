
def f():
    return 'string'

with f() as a:
    a = 3

a = {}
