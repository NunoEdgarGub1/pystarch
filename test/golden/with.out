a Dict(Unknown,Unknown) {}
f Function()

testcases/with.py:6 type-change "a = ..." (Str -> Num)
testcases/with.py:6 reassignment "a = ..."
